src/routes/landing/components/CityEntry.js

```diff
@@ -56,22 +70,34 @@ class CityEntry extends React.PureComponent<PropType> {
   render () {
     const { city, theme, filterText } = this.props
     const normalizedFilter = normalizeSearchString(filterText)
-    const aliases = this.getMatchingAliases(city, normalizedFilter)
+    let aliases = this.getMatchingAliases(city, normalizedFilter)
+    const sliceNeeded = aliases.length > MAX_NUMBER_OF_ALIASES_SHOWN
+    aliases = sliceNeeded ? aliases.slice(0, MAX_NUMBER_OF_ALIASES_SHOWN) : aliases
```

Comment:
From the docs of Array.prototype.slice(): `If end is greater than the length of the sequence, slice extracts through to the end of the sequence (arr.length).` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)

Therefore the following is enough:
```suggestion
    const aliases = this.getMatchingAliases(city, normalizedFilter).slice(0, MAX_NUMBER_OF_ALIASES_SHOWN)
    const sliceNeeded = aliases.length > MAX_NUMBER_OF_ALIASES_SHOWN
```